### PR TITLE
[prod-stable]Add order process submenu

### PR DESCRIPTION
**Pull request by lgalis**
Add the order processes nav menu.

Screenshot from 2020-11-17 00-18-29

### CODE REVIEW

**Review by eclarizio — APPROVED** [review]

File: main.yml [hunk 1]
```diff
@@ -379,7 +385,7 @@ openshift:
       - id: overview
         title: Overview
         group: openshift
-  source_repo: 
+  source_repo:
```
Comment (kruai) [inline main.yml]: Is this change intentional?

File: main.yml [hunk 2]
```diff
@@ -557,7 +563,7 @@ trust:
     paths:
       - /security/insights
   source_repo: https://github.com/RedHatInsights/trust-frontend
-  
+
```
Comment (kruai) [inline main.yml]: Is this change intentional?

**Review by kruai — COMMENTED** [review]
Looks good, in general (though there are some tiny changes which I'm curious about).
When do you want this released to prod-stable?

**Comment by lgalis** [comment]
Pushed to prod-stable today, please.

**Comment by kruai** [comment]
@lgalis OK. Can you please revert the changes you added that aren't related to order-processes? I'll merge it after that

**Comment by lgalis** [comment]
@kruai - done, thank you!

**Review by kruai — APPROVED** [review]
